bitnami/kubeapps/templates/apprepository/rbac.yaml

```
@@ -212,7 +219,7 @@ rules:
     verbs:
       - '*'
   - apiGroups:
-      - ""
+      - "kubeapps.com"
```

```suggestion
      - kubeapps.com
```
I have just checked that this change breaks our cypress test (placed under `.vib/kubeapps/cypress`) with the following issue.

```
Timed out retrying after 30000ms: Expected to find content: 'bitnamicharts' but never did. 
```
Could you please take a look? You can install the chart with the command below:

```
helm install kubeapps . -f ../../.vib/kubeapps/runtime-parameters.yaml
```

Removing this line makes the test work.